### ISSUE TITLE
New Nightfall music

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#412]
Probably a story music, by wansti. Available from his website:

* https://discarded-ideas.org/music
* Direct link: https://discarded-ideas.org/sites/discarded-ideas.org/files/music/windstille-nightfall.ogg

Licensed as CC-BY-SA 3.0.

Valete,
Quintus

**Comment by DarkAceZ** [comment]
Yeah, that would work nicely for story music.

**Comment by datahead8888** [comment]
By story music you mean tutorial level music, correct?
The tutorial probably will have to introduce the story concepts.
@danfun64 - do you have any thoughts on this song?

**Comment by Danfun64** [comment]
It sounds too supertux-esque. 
TSC's soundtrack as a whole seems less synthesized than supertux's. The only way that song could possibly work is as a remix.

**Comment by datahead8888** [comment]
My thought is that TSC is trying new styles out and that Super Tux as a game will eventually need a new composer other than just wansti.  Thus this will become less of a problem over time.

You're not the first person to bring up this concern, so we should probably not add this until there is more team discussion and hopefully some sort of agreement as to what we should do.

**Comment by datahead8888** [comment]
@sydneyjd, @Luiji, @Quintus - does this song fit the TSC game?  Does anyone have problems with including something originally meant for use in Super Tux?

**Comment by sydneyjd** [comment]
To put it bluntly, i could see this work as some sort of creidts theme or some dark level, but to me this doesn't fit TSC's 'style' very well. 

-Sydney

**Comment by lambda-11235** [comment]
@sydneyjd I think it would go good with a level in the heart of the ruins of the ancient civilization. Maybe somewhere where that is historically relevant or culturally important to the story. Vocals are usually very good for those times.

**Comment by sydneyjd** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Yes, it would work well in a level like that.

- -Sydney
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQEcBAEBCAAGBQJWr4aZAAoJELX8vPY0tOUs+d0H/jAIG1B3vybXM2eU5AXLTPvS
kXmgHnpTQdloMRxErhbEA4109xttQNBj+RkAB43NIN4d46s5Xfuxxbigg3/lQ2zR
3HMf1qdD8OEjQeau948ZlN5ll/3kKQ5xYTAIs2bTGAnjeCoZtTr5IUYLhyLcxcKw
QxIZiNEC+UB1sWoV+bo6mIJbmUWBB4iw53lYUKiEttB+9hErD5GhhwyeYNcDSgpA
uxGF/U8axd4yE4Y84l2TP6t+Dv81L8rITKOKI7XLKN0op2utuejLyaW5zkYZLUdf
tS2DpSZjXI+BoHXkdAMj6Qe3qDlV62tOgJ7Y1ppR3ZEXjh/ih21G7yGLMwHAVj4=
=A7sw
-----END PGP SIGNATURE-----

**Comment by sydneyjd** [comment]
Oops. ;)

**Comment by datahead8888** [comment]
In what release will we probably add this?

**Comment by datahead8888** [comment]
@Quintus, what do you think about adding this music to the game, given the prior discussion?

**Comment by Quintus** [comment]
I agree with lambda. I am okay with this song in the game, but it does not loop, which would need to be changed.

Vale,
Quintus
